addons/website/static/tests/tours/snippets_all_drag_and_drop.js

```diff
@@ -26,6 +26,13 @@ for (const snippet of snippetsNames) {
     }, {
         content: `click on 'BLOCKS' tab (${snippet})`,
         trigger: ".o_we_add_snippet_btn",
+        run: function (actions) {
+            // FIXME cannot find the reason why this setTimeout is needed to
+            // after reverting ab7508393376075f95d6dd5925e7f4462936d2, to check
+            // (this commit is however reverted temporarily until a better
+            // solution is found).
+            setTimeout(() => actions.auto(), 0);
```

Comment:
@rdeodoo Sorry, I introduce a `setTimeout` in one of your beloved test, fixing that deadlock is becoming urgent :grimacing:  I cannot figure out why it is needed but as I explained, this should only be temporary until I find out a better solution to reintroduce the purpose of the commit I am reverting here.